rbac/management/principal/view.py

```diff
@@ -43,8 +42,10 @@
 VALID_BOOLEAN_VALUE = ["true", "false"]
 USERNAME_ONLY_KEY = "username_only"
 PRINCIPAL_TYPE_KEY = "type"
+USER_KEY = "user"
+SA_KEY = "service-account"
 ALL_KEY = "all"
-VALID_PRINCIPAL_TYPE_VALUE = [Principal.Types.SERVICE_ACCOUNT, Principal.Types.USER, ALL_KEY]
+VALID_PRINCIPAL_TYPE_VALUE = [SA_KEY, USER_KEY, ALL_KEY]
```

Comment:
How about using str(Principal.Types.SERVICE_ACCOUNT)

Reply:
this change wont be included since it is already working fine